templates/easyappointments/0/rancher-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+catalog:
+  name: 'Easy!Appointments'
+  version: 0.1.0
```

Comment:
What do you think if you match this version with service version??

```
-  version: 0.1.0
+  version: 1.3.0
```

Reply:
Is that standard practice?

Reply:
Trying to make it standard due to it's more clear.. Package version `1.3.0` install service version `1.3.0`